cfme/tests/webui/test_advanced_search.py

```diff
@@ -257,3 +163,179 @@ def test_filter_crud(param, appliance):
     else:
         if view.my_filters.is_displayed:
             assert not view.my_filters.navigation.has_item(filter_name), "Filter wasn't deleted!"
+
+
+_tests = [_advanced_search_button_displayed, _can_open_advanced_search, _filter_crud]
+
+
+def methodized(metafunc):
```

Comment:
Please add docstrings here and to the function below. It took me some time to understand what's going on here :)